var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -3,185 +3,122 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-import os
 from spack import *
 
 
-class Neuron(Package):
+class Neuron(CMakePackage):
     """NEURON is a simulation environment for single and networks of neurons.
-
```

Comment:
Please keep this empty line. The short first paragraph is used in every place we need a short description of the software. Without the new line short and long description are the same.

Reply:
Done!